### PR TITLE
[js/web] Support WebGL for ort format models in benchmarks

### PR DESCRIPTION
**Pull request by hanbitmyths**
Benchmark models such as mobilenet and resnet50 are not working in ort format with higher optimization level. There are two main reasons,

- Missing FusedConv kernel
- Long type with unsigned number, which interpret -1 to invalid number

This PR fixes two issues and all benchmark models with ort format are working properly in WebGL.

### CODE REVIEW

File: js/web/lib/onnxjs/backends/webgl/op-resolve-rules.ts [hunk 1]
```diff
@@ -61,6 +61,10 @@ export const WEBGL_OP_RESOLVE_RULES: readonly OpSet.ResolveRule[] = [
   ['Exp', '', '6+', unaryOps.exp],
   ['Flatten', '', '1+', flatten, parseFlattenAttributes],
   ['Floor', '', '6+', unaryOps.floor],
+  // FusedConv is under com.microsoft domain, not under ai.onnx.
+  // Since there is no conflict yet, only a name is used here.
+  // But, when any name conflict occurs, using a domain needs to be considered.
+  ['FusedConv', '', '1+', conv, parseConvAttributes],
```
Comment (fs-eire) [inline js/web/lib/onnxjs/backends/webgl/op-resolve-rules.ts:67]: the second parameter is the opset domain.

File: js/package.json [hunk 2]
```diff
@@ -15,5 +15,6 @@
     "lint": "eslint . --ext .ts --ext .tsx",
     "format": "clang-format --glob=\"{common/lib/**/*.ts,node/{lib,script,test}/**/*.ts,node/src/**/*.{cc,h},web/{lib,script,test}/**/*.ts,react_native/{android,example,ios,lib}/**/*.{ts,mm,java}}\" --style=file -i"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {}
```
Comment (fs-eire) [inline js/package.json:19]: is it required by some toolset to add an entry for "dependencies"?

File: js/web/docs/operators.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ See [Compatibility](../README.md#Compatibility) for a list of the supported plat
 | [HardSigmoid](https://github.com/onnx/onnx/blob/master/docs/Operators.md#HardSigmoid) |  |
 | [HardSwish](https://github.com/onnx/onnx/blob/master/docs/Operators.md#HardSwish) |  |
 | [Hardmax](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Hardmax) |  |
-| [Identity](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Identity) | [1-12](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Identity-1), [13](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Identity-13), [14-15](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Identity-14), [16+](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Identity-16) |
+| [Identity](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Identity) | [1-12](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Identity-1), [13](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Identity-13), [14+](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Identity-14) |
```
Comment (fs-eire) [inline js/web/docs/operators.md:70]: this change should not appear here. did you try rebase/merge latest master?

File: js/web/lib/onnxjs/backends/webgl/ops/fuse-utils.ts [hunk 4]
```diff
@@ -36,11 +36,10 @@ export function getActicationSnippet(attributes: InternalActivationAttributes) {
 }
 
 export const parseInternalActivationAttributes = (attributes: Attribute): InternalActivationAttributes => {
-  const activation = attributes.getString('__internal_activation', '');
+  const activation = attributes.getString('activation', '');
 
   if (activation === 'Clip') {
-    const clipMax = attributes.getFloat('__clip_max', 3.402823e+38);
-    const clipMin = attributes.getFloat('__clip_min', -3.402823e+38);
+    const [clipMin, clipMax] = attributes.getFloats('activation_params', [-3.402823e+38, 3.402823e+38]);
```
Comment (fs-eire) [inline js/web/lib/onnxjs/backends/webgl/ops/fuse-utils.ts:42]: I remember there was a change to move those into const CLIP_MIN and CLIP_MAX. we can reuse them

File: js/web/lib/onnxjs/util.ts [hunk 5]
```diff
@@ -476,7 +476,7 @@ export class LongUtil {
     if (Long.isLong(n)) {
       return n.toNumber();
     } else if (n instanceof flatbuffers.Long) {
-      return Long.fromValue({low: n.low, high: n.high, unsigned: true}).toNumber();
+      return Long.fromValue({low: n.low, high: n.high, unsigned: false}).toNumber();
```
Comment (fs-eire) [inline js/web/lib/onnxjs/util.ts:479]: what is the change of this flag for?

File: js/package.json (hunk 2 above)
Comment (hanbitmyths) [reply js/package.json:19]: There was a warning message for caniuse-lite package and 'dependencies' is added by a npm automatically.

File: js/web/lib/onnxjs/backends/webgl/ops/fuse-utils.ts (hunk 4 above)
Comment (hanbitmyths) [reply js/web/lib/onnxjs/backends/webgl/ops/fuse-utils.ts:42]: good catch

File: js/web/lib/onnxjs/util.ts (hunk 5 above)
Comment (hanbitmyths) [reply js/web/lib/onnxjs/util.ts:479]: When ONNX model is converted to ORT model and there is no axis attribute in softmax, a tool will add an attribute 'axis' with a default value -1 into a model. floatbuffers.Long representation of int64 -1 is {low: -1, high: -1}. If 'unsigned' is true at Long type, a value will be 18446744073709552000, not -1.

**Review by fs-eire — APPROVED** [review]

**Pull request merged**